crawler/http/crawler/settings.py

```diff
@@ -16,6 +16,9 @@
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
 USER_AGENT = "sycamore crawler (+https://github.com/aryn-ai/sycamore)"
 
+# Crawl at about 1 QPS
+DOWNLOAD_DELAY = 1
```

Comment:
Where is this used?

Reply:
Magically inside of scrapy, same as pretty much everything else in the config file.  I tested it and it was about 1/second.
https://docs.scrapy.org/en/latest/topics/settings.html#download-delay